xbmc/music/MusicDatabase.cpp

```diff
@@ -3925,8 +3966,29 @@ void CMusicDatabase::UpdateTables(int version)
           " strType, iRating, lastScraped, "
           " strReleaseType"
           " FROM album");
-      m_pDS->exec("DROP TABLE album");
-      m_pDS->exec("ALTER TABLE album_new RENAME TO album");
+    m_pDS->exec("DROP TABLE album");
+    m_pDS->exec("ALTER TABLE album_new RENAME TO album");
+
+    //Remove strJoinPhrase, boolFeatured from song_artist table and add idRole
```

Comment:
as  @evilhamster pointed out this part below needs to be in a new db schema version